app/src/main/java/com/jerboa/ui/theme/Type.kt

```diff
@@ -1,44 +1,76 @@
 package com.jerboa.ui.theme
 
 import androidx.compose.material3.Typography
-import androidx.compose.ui.unit.TextUnit
+import androidx.compose.ui.unit.sp
 
-const val FONT_LARGE_MULTIPLIER = 1.3
-const val FONT_MEDIUM_MULTIPLIER = 0.8
-const val FONT_SMALL_MULTIPLIER = 0.7
+const val FONT_SMALL_ADD = -7
+const val FONT_MEDIUM_ADD = -3
+const val FONT_LARGE_ADD = 0
+const val LINE_HEIGHT_MULTIPLIER = 1.5
 
 // Set of Material typography styles to start with
-fun generateTypography(baseFontSize: TextUnit): Typography {
+// Example from here: https://egeniq.com/blog/dynamic-font-sizes-with-jetpack-compose/
```

Comment:
I was incorrectly using multipliers on the `baseFontSize`. The link above suggested to use simple additions, and a multiplier for line height.